backend/src/modules/tournament-tree/core/domain/TournamentTree.ts

```diff
@@ -58,7 +70,23 @@ export class TournamentTree {
     return this.tournamentTreeArray.filter((item) => item.firstTeam && item.secondTeam).map((item) => item.fightingTeamsGroupId.raw);
   }
 
-  public finishMatch(matchId: string, winnerId: string): void {}
+  public getMatchIdFromMatchNumberInSequence(matchNumberInSequence: number): string | undefined {
+    return this.tournamentTreeArray.find((match) => match.matchNumberInSequence === matchNumberInSequence)?.fightingTeamsGroupId.raw;
+  }
+
+  public setMatchWinner(matchId: string, winnerId: string) {
```

Comment:
Zastanawiam się skąd u Ciebie to przyzwyczajenie do metod z przedrostkiem `get` / `set`. To straszna zaszłośc z tzw. standardu Java Beans, co było kiedyś potrzebne do poprawnego działania programu. Teraz się staramy z tym walczyć, bo to "zwalnia z myślenia" i robi model anemiczny, czyli prawie bez reguł, a jednak reguły u Ciebie są :) 

Co by się stało jakbyś usunął get z nazw metod :) ? 
A z tej tutaj set, może jest lepsza nazwa - mniej techniczna a bardziej biznesowa, na tą operację? Jak się mówi na turnieju 'ustaw zwyciezce meczu`? Na pewno nie :) Zazwyczaj chyba `zapisz wynik meczu`, `zapisz kto wygrał mecz`.

Reply:
hmmm, ciekawi mnie to, ze o tym wspominasz. Nie sądziłem, ze może być to aż tak ważne. Hmmm...
Dobrze wiedzieć, że to "nie jest elegancko".
W takim razie mógłbyś mi napisać (tego potrzebuje) jakie zalety ma użycie bardziej biznesowego nazewnictwa?
Czy również chodzi o to, że tak technicznie mamy problemy przy zmianie nazewnictwa?

Reply:
Pierwsza odpowiedź na stackoverflow w tym pytaniu powinna pomóc:
https://stackoverflow.com/questions/46139501/should-getters-from-domain-classes-not-be-prefixed-with-get-in-ddd-and-why

Napisz czy coś CI to dało :) 
Chodzi o to, żebyś po prostu nie "kodował" ale pisał. Tak, żeby jak ktoś przeczyta, kto gra np. w piłkarzyki wiedział, o czym jest kod. A nie żeby tłumaczyć z piłkarzykowego na programistyczne.
Miałem tak nie raz w projektach, że były np. 3 różne nazwy na to samo. 
Czyli trzeba się trzymać w miarę tego, jeśli się da, jak mówi człowiek. Szczególnie w warstwie domain - bo tutaj nie ma kwestii technicznych. Próbuj czytać kod (publiczne API domain) jak tekst i patrz czy to wygląda płynnie.